app/api/ada/lib/state-fetch/remoteFetcher.js

```diff
@@ -36,13 +36,26 @@ const backendUrl = CONFIG.network.backendUrl;
  * https://github.com/Emurgo/yoroi-backend-service/
  */
 export class RemoteFetcher implements IFetcher {
+
+  lastLaunchVersion: () => string;
+  currentLocale: () => string;
+
+  constructor(lastLaunchVersion: () => string, currentLocale: () => string) {
+    this.lastLaunchVersion = lastLaunchVersion;
+    this.currentLocale = currentLocale;
+  }
+
   getUTXOsForAddresses = (body: AddressUtxoRequest): Promise<AddressUtxoResponse> => (
     axios(
       `${backendUrl}/api/txs/utxoForAddresses`,
       {
         method: 'post',
         data: {
           addresses: body.addresses
+        },
+        headers: {
+          'x-yoroi-version': this.lastLaunchVersion(),
```

Comment:
I just found out that the `X-` convention got deprecated back in 2012 https://tools.ietf.org/html/rfc6648. It’s recommended to not use it.

Reply:
:O I see. The reason I used this naming is that there seem to be other parameters with the same naming configured in the server.  
![allow_headers](https://user-images.githubusercontent.com/1937074/60228412-9f652880-9860-11e9-8443-8b9106e747bf.png)